dto/api_input.py

```diff
@@ -121,14 +123,64 @@ class InputUserLoginDTO(BaseDTO):
 class UpdateUserPreferencesDTO(BaseDTO):
     preferences = fields.List(fields.Int, required=True)
     user_id = fields.Int(required=False, missing=None)
+    type = fields.Str(required=False, missing=None)
 
 
 class GetUserPreferencesDTO(BaseDTO):
     user_id = fields.Int(required=False, missing=None)
+    type = fields.Str(required=False, missing=None)
 
 
 class InputUserByGoogleDTO(BaseDTO):
     email = fields.Str(required=True)
     id = fields.Str(required=False, missing=None)
     verified_email = fields.Bool(required=False, missing=None)
-    picture = fields.Str(required=False, missing=None)
\ No newline at end of file
+    picture = fields.Str(required=False, missing=None)
+
+
+class SportPreferenceDTO:
+    def __init__(self):
+        self.main_table = "UserPreference"
+        self.related_table = "Sport"
+        self.user_id_field = "users_id"
+        self.type_id_field = "sports_id"
+        self.related_name = "sport_name"
+        self.related_logo = "sport_img"
+        self.related_id = "sport_id"
+
+
+class TeamPreferenceDTO:
+    def __init__(self):
```

Comment:
We should not expose our database structure on API level. I doubt our API cares where it gets it data from from table A or Table B. That is for BL and DAl to play around with